server/src/jira/session-setup.ts

```diff
@@ -0,0 +1,31 @@
+import session from "express-session";
+import pgSession from "connect-pg-simple";
+import pg from "pg";
+import { ConfigService } from "src/config/config.service";
+
+export function createSession(configService: ConfigService) {
+  const { username, password, host, port, name } = configService.config.database;
+  const pgPool = new pg.Pool({
+    database: name,
+    user: username,
+    password,
+    host,
+    port,
+  });
+
+  return session({
+    store: new (pgSession(session))({
+      pool: pgPool,
+      createTableIfMissing: true,
```

Comment:
I would be inclined to use `createTableIfMissing` only in local development environments like we do with TypeORM's `synchronize` and add a database migration for production.

While I can find no mention about this in `connect-pg-simple`'s docs, it appears that [there have been problems](https://github.com/voxpelli/node-connect-pg-simple/issues/200) with the `createTableIfMissing` option earlier. It would also seem prudent to use our migration mechanism for this since we have one.